### DISCUSSION TITLE
最新特性一些设计上的相关讨论

### DISCUSSION THREAD

**Discussion by ted423** [ronggang/PT-Plugin-Plus#1248] [General]
![image](https://user-images.githubusercontent.com/7042766/207553336-1f43e2e8-2966-42a2-8067-3628d187b1e0.png)
魔力值积分这一列，个人感觉可以加个title 说明哪个是魔力值哪个是积分

![image](https://user-images.githubusercontent.com/7042766/207553517-75dcd9c0-5b9b-4633-9d5e-81c9ae7a9411.png)
这个图标我感觉单独列出来加个title "待升级差距" 

还有就是上传下载箭头或许也可以统一一下
![image](https://user-images.githubusercontent.com/7042766/207553769-3f0e78c0-a3b5-4ba8-aaa8-e1e7e2b543b1.png)
![image](https://user-images.githubusercontent.com/7042766/207553800-e53bd722-9567-456b-984a-5d07e7acf81a.png)

**Comment by ted423** [comment]
或者等级也可以弄个图标
这样和![待升级](https://user-images.githubusercontent.com/7042766/207553517-75dcd9c0-5b9b-4633-9d5e-81c9ae7a9411.png)
可以对齐

**Reply by fzlins** [reply]
> 或者等级也可以弄个图标 这样和![待升级](https://user-images.githubusercontent.com/7042766/207553517-75dcd9c0-5b9b-4633-9d5e-81c9ae7a9411.png) 可以对齐

这个怎么样![image](https://user-images.githubusercontent.com/100051558/207609288-40a10f7a-e6bb-4708-9280-2878e990bbfa.png)

上下箭头用哪种比较好

**Reply by ted423** [reply]
其实我都行，统一了就行.

**Comment by zhuweitung** [comment]
之前海胆加了一个等级积分，和积分同一个图标
![image](https://user-images.githubusercontent.com/20264596/207554975-982865e8-26cd-45d3-b7e4-fe3a262d4a8d.png)
是否其他类型的积分也使用同一图标，还是设置不同图标

**Reply by mitlearn** [reply]
> 之前海胆加了一个等级积分，和积分同一个图标
> ![image](https://user-images.githubusercontent.com/20264596/207554975-982865e8-26cd-45d3-b7e4-fe3a262d4a8d.png)
> 是否其他类型的积分也使用同一图标，还是设置不同图标

这里的图标选取参考了u3d(采用一套现成的方案比重新定义好一些) 所以产生了冲突
![8E821BDF-B44C-4F78-AA12-5BF8A4383015](https://user-images.githubusercontent.com/64004444/207556019-15755714-398c-45ef-b487-517459280b58.jpeg)

个人建议站点图标与此冲突时采用其他颜色替代
